.ci/static-checks.sh

```diff
@@ -284,13 +314,49 @@ check_docs()
 
 	info "Checking documentation"
 
-	local docs=$(find . -name "*.md" |grep -v "vendor/" || true)
-
+	local doc
+	local docs
+	local docs_status
+	local new_docs
+	local new_urls
 	local url
+
+	if [ "$master_branch" = "true" ]
+	then
+		# Check all documents
+		docs=$(find . -name "*.md" |grep -v "vendor/" || true)
```

Comment:
nit: missing ` ` before the `|` looks odd to me. And others below.

Reply:
Fixed.